app/models/shipit/stack.rb

```diff
@@ -58,6 +58,10 @@ def self.repo(full_name)
       Repository.find_by(owner: repo_owner, name: repo_name).stacks
     end
 
+    def self.not_archived
```

Comment:
I know we're modeling behavior of _archival_ and we've been using that nomenclature in our stories, but had you considered language which provides a clear affirmative/negative case; `active` vs `inactive` perhaps? I can't think of a word to describe something that _isn't_ archived other than `unarchived` which - in all the ambiguity of the English language is more likely to mean an object which was _once_ archived, but is no longer.

Inverted attribute names tend to trip me up when reading code - in general, I find it much easier to grok what _is_ rather than what _isn't_ - especially when we start applying additional predicates.

Reply:
I could do hidden/visible if you think that’s better.

Reply:
also, this could be turned into a scope

Reply:
Scopes are an anti-pattern.

Reply:
Not sure what you mean by that, but regardless they're used across the codebase, so might as well stay consistent.

Reply:
Done. I can explain later in more detail, but scopes are pointless at a minimum. Anywho, it’s now a scope :)